src/components/Block/Block.tsx

```diff
@@ -83,15 +83,19 @@ export default class Block extends React.Component<BlockProps> {
     const { children, textSize } = this.props;
 
     return (
-      <ClickableContext.Consumer>
-        {clickableContext => (
-          <BlockContext.Provider value={{ textSize }}>
-            <div className={this.getClasses(clickableContext.withinClickable)}>
-              <div className={`y-block--inner ${mergeStyles(getInnerStyles(this.props))}`}>{children}</div>
-            </div>
-          </BlockContext.Provider>
+      <TextSizeContext.Consumer>
```

Comment:
Added consumer here, so that the context could be passed along, if `textSize` is not set in the current instance.